frontend/model/contracts/identity.js

```diff
@@ -121,15 +123,22 @@ sbp('chelonia/defineContract', {
     'gi.contracts/identity/createDirectMessage': {
       validate: (data, { state, getters }) => {
         objectOf({
-          privacyLevel: unionOf(...Object.values(CHATROOM_PRIVACY_LEVEL).map(v => literalOf(v))),
-          contractID: string
+          // NOTE: 'groupContractID' is the contract ID of the group where the direct messages is created
+          //       it's optional parameter meaning the direct message could be created outside of the group
+          groupContractID: optional(string),
```

Comment:
I'm not sure about this `groupContractID`. It mostly seems unnecessary based on how I understand DMs to work, which isn't tied to a group.

Reply:
@corrideat so this was added as a hint to help the UI keep track of DMs that were created in the context of a group. It's not really used for anything else except for that (helping visually organize DMs).